tasks/sign-index-image/sign-index-image.yaml

```diff
@@ -4,117 +4,86 @@ kind: Task
     tekton.dev/tags: release
 spec:
   description: >-
     Task to create an internalrequest to sign a FBC Index Image.
   params:
-    - name: requestJsonResults
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
       type: string
-      default: "{}"
-      description: The JSON results of the Internal Request
-    - name: targetIndex
+      default: "data.json"
+    - name: request
       type: string
-      description: targetIndex signing image
-    - name: requestUpdateTimeout
+      description: Signing pipeline name to handle this request
+      default: "hacbs-signing-pipeline"
+    - name: referenceImage
       type: string
-      default: "360"
-      description: Max seconds waiting for the status update
-    - name: signingPipelineImage
+      description: The image to be signed.
+    - name: manifestDigestImage
       type: string
-      default: "quay.io/redhat-isv/operator-pipelines-images:released"
-      description: An image with CLI tools needed for the signing
+      default: ""
+      description: Manifest Digest Image used to extract the SHA
     - name: requester
       type: string
-      value: ""
-      description: Name of the user that requested the signing, for auditing purposes
-    - name: configMapName
+      description: The request user
+    - name: requestTimeout
       type: string
-      description: ConfigMap Name for this request
-    - name: pipelineRunName
-      type: string
-      description: The name of the Parent PipelineRun of this task
-  results:
-    - name: requestMessage
-      type: string
-      description: Internal Request message
-    - name: requestReason
-      type: string
-      description: Internal Request reason
-    - name: requestResults
-      type: string
-      description: Internal Request results
+      default: "180"
+      description: InternalRequest timeout
+  workspaces:
+    - name: data
+      description: workspace to read and save files
   steps:
     - name: sign-index-image
       image:
-        quay.io/hacbs-release/release-utils@sha256:5733ece907aa70d6ebced36484c936731e8b27bfcf87fed226a0ecf95047a6b8
+        quay.io/hacbs-release/release-utils:6e92a6f8df8ef1cbecfb4c25b73ec6d92bded527
       script: |
         #!/usr/bin/env sh
-        PATH=/bin:/usr/bin:/usr/local/bin
-        export PATH
+        #
+        set -e
 
-        jsonInputFile="/tmp/$$.json"
-        cat > ${jsonInputFile} <<JSON
-        $(params.requestJsonResults)
-        JSON
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
 
-        internalIndexImageCopyResolved=`jq -cr .jsonBuildInfo ${jsonInputFile} \
-        | jq -cr .internal_index_image_copy_resolved`
+        # The signing configuration should be set as data.sign in the releaseAdmissionPlan. Previously it used to be
```

Comment:
This documentation is good. Not sure it is the best spot for it. Maybe in the README before the parameters for now?

Reply:
Moved this to the README